riscv/processor.cc

```diff
@@ -310,6 +315,26 @@ void processor_t::parse_isa_string(const char* str)
       extension_table[EXT_SVPBMT] = true;
     } else if (ext_str == "svinval") {
       extension_table[EXT_SVINVAL] = true;
+    } else if (ext_str == "zfhmin") {
+      if (extension_table[EXT_ZFINX]) {  //z[fdq]inx_zfhmin
+        extension_table[EXT_ZHINX] = true;
+        extension_table[EXT_ZHINXMIN] = true;
+      } else {
+        if (!((max_isa >> ('f' - 'a')) & 1))
+          bad_isa_string(str, "Base 'Zfhmin' extension requires 'F'. Or Zfinx version of 'Zfhmin' should be 'Z[FDQ]inx_zfhmin'");
+        extension_table[EXT_ZFHMIN] = true;
+      }
+    } else if (ext_str == "zfinx") {
+      extension_table[EXT_ZFINX] = true;
+    } else if (ext_str == "zdinx") {
+      extension_table[EXT_ZDINX] = true;
+      extension_table[EXT_ZFINX] = true;
+    } else if (ext_str == "zqinx") {
+      extension_table[EXT_ZQINX] = true; //not work currently
```

Comment:
One last nit: if Zqinx doesn't work, this should give a reasonable error message, instead of silently doing the wrong thing.

Reply:
Ok, Zqinx flag is defined at the beginning of the v0.41 spec, However, there is no further description about it. Maybe we can just delete this flag or report message "Zqinx is not supported currently." here.

Reply:
Just delete this case.  No need to have a special error message Zqinx--it should issue the same error message as any other extension Spike doesn't implement.

Reply:
OK.

Reply:
OK. Thanks.